catkin_ws/src/state_estimation/src/simple_state_estimation.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+import rospy
+import math
+import numpy as np
+
+from geometry_msgs.msg import Point, Vector3, Quaternion
+from sensor_msgs.msg import Imu, FluidPressure
+
+from aquadrone_msgs.msg import SubState
+
+class StateEstimator:
```

Comment:
What sort of state estimation is this?
From what I can tell, it seems like the position and velocity variances are going to blow up to infinity.
Also, the increment dynamics function should (eventually) take input as to what thrust is being applied by the motors (and torque from buoyancy).

Reply:
This is a very simple version, that was just meant to get the basics functioning.

The Kalman Filter version that Im working on in another branch (and will probably merge in to replace this) will be using the thruster forces, and likely have damping to prevent velocity from blowing up.

Position will blow up until we have a way to directly measure it.